runelite-client/src/main/java/net/runelite/client/util/GameEventManager.java

```diff
@@ -195,4 +196,12 @@ public void simulateGameEvents(Object subscriber)
 			eventBus.unregister(subscriber);
 		});
 	}
+
+	public static Boolean shouldClearSpawns(GameStateChanged gameStateChanged)
```

Comment:
This should be named such that it's clear that it is referencing NPC spawns specifically.